TUTORIAL.md

```diff
@@ -86,8 +86,12 @@ with this language extension enabled.
 
 There are a few important caveats to mention about this library:
 
-* By default, any non-2XX status code response results in a runtime exception.
-  See the examples of `setRequestIgnoreStatus` below for more information
+* By default, any non-2XX status code response results in a runtime
```

Comment:
This is obviously a good change, but I'd like to suggest making it slightly better: change this to say that by default status code will not result in runtime exceptions, contrary to previous behavior of the library.